Makefile

```diff
@@ -3,12 +3,12 @@ config ?= release
 PACKAGE := peg
 GET_DEPENDENCIES_WIITH := corral fetch
 CLEAN_DEPENDENCIES_WIITH := corral clean
-COMPILE_WITH := corral run -- ponyc
+COMPILE_WITH := corral run -- ponyc -V1
 
 BUILD_DIR ?= build/$(config)
 SRC_DIR := $(PACKAGE)
 EXAMPLES_DIR := examples
-tests_binary := $(BUILD_DIR)/$(PACKAGE)
```

Comment:
why this change? it would make the peg library different than our others. I'd prefer to keep them the same unless there's a strong reason for the change.

Reply:
Ok, I've reverted that bit